app/views/admin/action_pages/edit_partners.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for(:nav) { render "action_page_nav" } %>
+
+<h2>Partners for <%= @actionPage.title %></h2>
+
+<p> Add partner logos and email signups to this action. </p>
+
+<%= form_for([:admin, @actionPage]) do |f| %>
```

Comment:
Just noticed while testing this that #755 broke the update action for forms like this -- the call to `institutions_params` assumes that the request is made from the main form

```ruby
  def update
  ...
      @actionPage.update_attributes(action_page_params)
      if (institutions_params[:reset] && institutions_params[:reset] == "1") ||
  ...
  end

  def institutions_params
    params.require(:institutions).permit(%i(category reset))
  end
```
```